specifyweb/frontend/js_src/lib/localization/query.ts

```diff
@@ -638,4 +638,7 @@ export const queryText = createDictionary({
       {maxLength:number|formatted}
     `,
   },
+  hiddenFieldsMess: {
```

Comment:
@CarolineDenis this is not a good name for a localization string. Since this pull request is already merged, can you create a new pull request with a fix for this?